### PR TITLE
feat(devops): Add deploy and code review pipelines

### PR DESCRIPTION
**Pull request by pietro-tota**
### List of changes
Add deploy and code review pipelines for `pagopa-ecommerce-helpdesk-service` project

### Motivation and context
Add deploy and code review pipelines for `pagopa-ecommerce-helpdesk-service` project

### Type of changes

- [x] Add new pipeline
- [ ] Update pipeline configuration
- [ ] Remove pipeline

### :warning: If it's new pipeline with code review have you added pagopa-github-bot -> Role: admin?

- [x] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  # deploy secrets
+  pagopa-ecommerce-helpdesk-service-variables_secret_deploy = {
+    git_mail     = module.secrets.values["azure-devops-github-EMAIL"].value
+    git_username = module.secrets.values["azure-devops-github-USERNAME"].value
+    tenant_id    = module.secrets.values["TENANTID"].value
+  }
+}
+
+module "pagopa-ecommerce-helpdesk-service_code_review" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_code_review?ref=v2.2.0"
```
Comment (diegolagospagopa) [inline azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf:78]: please use the last version of the module

File: azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    prod_container_registry_name         = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_prod_name
+
+    # aks section
+    dev_kubernetes_service_conn  = azuredevops_serviceendpoint_kubernetes.aks_dev.id
+    uat_kubernetes_service_conn  = azuredevops_serviceendpoint_kubernetes.aks_uat.id
+    prod_kubernetes_service_conn = azuredevops_serviceendpoint_kubernetes.aks_prod.id
+
+    dev_container_namespace  = "pagopadcommonacr.azurecr.io"
+    uat_container_namespace  = "pagopaucommonacr.azurecr.io"
+    prod_container_namespace = "pagopapcommonacr.azurecr.io"
```
Comment (diegolagospagopa) [inline azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf:66]: use locals

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

File: azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf (hunk 1 above)
Comment (pietro-tota) [reply azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf:78]: Left this module version for two reason:

1.  align module used by this ecommerce service to the other ones
2. latest version will not override trigger with this configuration

So I prefer to left this version in order to have the same behavior as for the other ecommerce services

File: azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf (hunk 2 above)
Comment (pietro-tota) [reply azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf]: Fixed with https://github.com/pagopa/pagopa-azure-devops/pull/228/commits/368669af69c7843016703b4b83f5fe7d80907d00

**Review by diegolagospagopa — APPROVED** [review]

**Comment by pietro-tota** [comment]
@diegolagospagopa ping :) this PR has been already applied, has only to be merged.

**Review by infantesimone — APPROVED** [review]

**Pull request merged**